README.md

```diff
@@ -1,19 +1,81 @@
-# Readme (WIP)
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/146856/124335690-fc7ecd80-db4f-11eb-93fa-dcf4469bb07b.png" alt="Apollo GraphQL"/>
+</p>
 
-## Tuist
+<p align="center">
+  <a href="https://github.com/apollographql/apollo-ios-dev/actions/workflows/ci-tests.yml">
```

Comment:
Note that this badge for CI Test status on main wont display/work correctly until we have merged the `project-breakup` branch into `main` since main currently has no tests etc to run and display.